src/supaplex.c

```diff
@@ -1939,7 +1948,11 @@ void readConfig() //  proc near       ; CODE XREF: start:loc_46F0Fp
     FILE *file = openWritableFile("SUPAPLEX.CFG", "rb");
     if (file == NULL)
     {
+#ifdef __NDS__
+        if (errno == ENOENT || errno == ENOSYS)

```

Comment:
Two questions about this:
1. Why do you need to check `ENOSYS` for NDS exactly?
2. Would it be a problem if you kept that check for all platforms and not only NDS?

Reply:
1. This occurs when no SD card is available, resulting in the current working directory pointing to the read-only `nitro:/` directory. This can be easily reproduced when running in an emulator. It's probably a bug in libfilesystem, but I don't know how to fix it.
2. I'm not sure. What's your opinion on this?

Reply:
I think we can leave that for all platforms, yeah. To be honest, that `errno == ENOENT` check comes from the original assembly code and I never bothered to rethink it. I have no problem with leaving the additional `ENOSYS` check you added for all platforms and remove the `#ifdef` stuff 😄

Reply:
OK, done.